renderer/brave_render_thread_observer.h

```diff
@@ -27,6 +27,8 @@ class BraveRenderThreadObserver
   BraveRenderThreadObserver();
   ~BraveRenderThreadObserver() override;
 
+  bool is_tor_process() { return is_tor_process_; }
```

Comment:
const

Reply:
fixed in 2bf1be60fb4efbe59956a565c4d4d65824cae9f1